### ISSUE TITLE
Pyhton 3

### ISSUE THREAD

**Issue by NoOneAtHome** [marekrei/smuploader#9]
Any Plans to update to Python 3?

**Comment by marekrei** [comment]
Yes. I started a branch for it but ran into some difficulties, as I want to support python 2 also. 
Working on it.

**Comment by NoOneAtHome** [comment]
> Yes. I started a branch for it but ran into some difficulties, as I want to support python 2 also.
> Working on it.

Awesome, looking forward to it.
